rpcserver.go

```diff
@@ -735,7 +735,7 @@ func handleCreateRawSStx(_ context.Context, s *rpcServer, cmd interface{}) (inte
 		}
 
 		prevOut := wire.NewOutPoint(txHash, input.Vout, input.Tree)
-		txIn := wire.NewTxIn(prevOut, wire.NullValueIn, []byte{})
+		txIn := wire.NewTxIn(prevOut, input.Amt, []byte{})
```

Comment:
There is no reason to create a `[]byte{}` here either.  Passing `nil` would save an unnecessary allocation.  I know it was already that way, but since you're updating this anyway, mind making it `nil`?

Reply:
How are you so fast...